src/extension/ncwms/src/test/java/au/org/emii/geoserver/wms/LayerDescriptorTest.java

```diff
@@ -0,0 +1,25 @@
+package au.org.emii.geoserver.wms;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+
+public class LayerDescriptorTest extends TestCase {
+    @Test
+    public void testSimpleParsing() throws Exception {
+        LayerDescriptor layerDescriptor;
+
+        layerDescriptor = new LayerDescriptor("imos:some_layer/sea_water_velocity");
+        assertEquals("imos", layerDescriptor.workspace);
+        assertEquals("some_layer", layerDescriptor.layer);
```

Comment:
Lets think of better names then. I just think that having it as `groupLayerName` and `layerName` is just as confusing.